appcues/src/main/java/com/appcues/ui/composables/AppcuesComposition.kt

```diff
@@ -69,6 +71,21 @@ private fun MainSurface() {
                     viewModel.dismiss()
                 }
             }
+
+            val experienceState = LocalExperienceCompositionState.current
+            LaunchedEffect(state.value) {
+                state.value.let { uiState ->
+                    if (uiState is Rendering) {
+                        // trigger showing of content
+                        experienceState.isContentVisible.targetState = true
+                        experienceState.isBackdropVisible.targetState = true
+                    } else {
+                        // trigger dismissal of content
+                        experienceState.isContentVisible.targetState = false
+                        experienceState.isBackdropVisible.targetState = false
+                    }
+                }
```

Comment:
this ensures that we get one and only one content visibility update to trigger the animation start